src/services/nodeService.ts

```diff
@@ -26,9 +31,14 @@ class NodeService {
         ) as INodeBase[];
     }
 
-    public static updateNode = async (node: INode) => {
+    public static updateNode = async (node: INode, links: ILink[]) => {
+        const request: INodeSavingModel = {
+            node,
+            links,
+        };
+
         const apiClient = new ApiClient();
-        await apiClient.updateObject(endpoints.NODE, node);
+        await apiClient.updateObject(endpoints.NODE, request);
```

Comment:
I can change the variable name to requestBody. I find it useful to type this variable, since we are listening to an identical model in the backend. In my opinion, it reduces the risk slightly to have it typed. I know it is not enforced, but it is more visible what the backend expect.